frontend/pages/SoftwarePage/components/AddSoftwareAdvancedOptions/AddSoftwareAdvancedOptions.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
```

Comment:
The corresponding `index.ts` says it's an empty file
[frontend/pages/SoftwarePage/components/AddSoftwareAdvancedOptions/index.ts](https://github.com/fleetdm/fleet/pull/18575/files#diff-9b046acc60dda892773234c5ea39a3e1d8ae7fd52c28e9262a8e8c65d044dfa1)

Reply:
thanks. updated this